src/EnergyPlus/DesiccantDehumidifiers.cc

```diff
@@ -224,32 +224,31 @@ namespace DesiccantDehumidifiers {
-        Real64 CoilBypassedFlowFrac;        // Bypass air fraction for multimode DX coils
-        Array1D_string Alphas;              // Alpha input items for object
-        Array1D_string cAlphaFields;        // Alpha field names
-        Array1D_string cNumericFields;      // Numeric field names
-        Array1D<Real64> Numbers;            // Numeric input items for object
-        Array1D_bool lAlphaBlanks;          // Logical array, alpha field input BLANK = .TRUE.
-        Array1D_bool lNumericBlanks;        // Logical array, numeric field input BLANK = .TRUE.
-        bool errFlag;                       // local error flag
-        std::string RegenCoilType;          // Regen heating coil type
-        std::string RegenCoilName;          // Regen heating coil name
-        int SteamIndex;                     // steam coil Index
-        bool RegairHeatingCoilFlag = false; // local error flag
+        int DesicDehumNum;              // Current desiccant dehumidifier number
+        int NumAlphas;                  // Number of Alphas for each GetObjectItem call
+        int NumNumbers;                 // Number of Numbers for each GetObjectItem call
+        int IOStatus;                   // Used in GetObjectItem
+        bool ErrorsFound(false);        // Set to true if errors in input, fatal at end of routine
+        bool ErrorsFound2(false);       // Set to true if errors in input, fatal at end of routine
+        bool ErrorsFoundGeneric(false); // Set to true if errors in input, fatal at end of routine
+        bool IsNotOK;                   // Flag to verify name
+        bool OANodeError;               // Flag for check on outside air node
+        std::string RegenCoilInlet;     // Desiccant system regeneration air heater inlet node
+        std::string RegenCoilOutlet;    // Desiccant system regeneration air heater outlet node
+        int DesuperHeaterIndex;         // Index of desuperheater heating coil
+        int RegenCoilControlNodeNum;    // Control node number of regen heating coil
+        Real64 CoilBypassedFlowFrac;    // Bypass air fraction for multimode DX coils
+        Array1D_string Alphas;          // Alpha input items for object
+        Array1D_string cAlphaFields;    // Alpha field names
+        Array1D_string cNumericFields;  // Numeric field names
+        Array1D<Real64> Numbers;        // Numeric input items for object
+        Array1D_bool lAlphaBlanks;      // Logical array, alpha field input BLANK = .TRUE.
+        Array1D_bool lNumericBlanks;    // Logical array, numeric field input BLANK = .TRUE.
+        bool errFlag;                   // local error flag
+        std::string RegenCoilType;      // Regen heating coil type
+        std::string RegenCoilName;      // Regen heating coil name
+        int SteamIndex;                 // steam coil Index
+        bool RegairHeatingCoilFlag;     // local error flag
 
         auto &MaxNums(state.dataDesiccantDehumidifiers->MaxNums);
```

Comment:
This is a bad use of a reference shortcut. Also, I am not sure why these need to be in `state` and can't just be loal variables.

Reply:
This is not used for anything.

![image](https://user-images.githubusercontent.com/5778156/230252702-47631ec5-b00d-44d6-a372-93d8082ef349.png)

Reply:
All 3 of these were in state ... were.

        int MaxNums = 0;
        int MaxAlphas = 0;
        int TotalArgs = 0;